docs/streams/developer-guide/dsl-api.html

```diff
@@ -3691,6 +3692,10 @@ <h5><a class="toc-backref" href="#id16">KTable-KTable Foreign-Key
             Kafka Streams comes with a <code>test-utils</code> module to help you test your application <a href="testing.html">here</a>.
             </div>
         </div>
+        <div class="section" id="#naming-a-streams-app">
+             <a class="headerlink" href="#naming-a-streams-app" title="Permalink to this headline"><h2>Naming a Streams DSL application</h2></a>
+             Kafka Streams allows you to <a href="dsl-topology-naming.html">name processors</a> created via the Streams DSL
```

Comment:
req: does it make sense for this section to go after the "Testing a Streams Application" section? I would keep it together with the other DSL operators
prop: do you think we should include a quick line about why you'd want to name things here? Just wondering if we should protect against people who won't want to read the full manual, and won't realize that naming is essential for compatibility

Reply:
I think it's a bit subjective.  IMHO it should come after all the other operations are described as this is a "meta-operation".  At any rate,  I've moved it up closer so it's the first item in the list after describing the other operations.

Reply:
Fair enough